tests/basics/TrickAssignments32.py

```diff
@@ -1047,4 +1047,172 @@ def __bytes__(self):
     print("Ouch.!")
 
 
+def someFunctionThatReturnsDeletedValueViaEq():
```

Comment:
Can you do the same to `__lt__`, `__lte__`, `__gt__`, `__gte__`, `__neq__` (spelling ?) too?

Yours,
Kay

Reply:
Sure, I will do this